docs/rules/no-extra-bind.md

```diff
@@ -1,60 +1,85 @@
 # Disallow unnecessary function binding (no-extra-bind)
 
-Function binding makes no difference in function expressions,
-when the function does not use `this`.
+The `bind()` method is used to create functions with specific `this` values and, optionally, binds arguments to specific values. When used to specify the value of `this`, it's important that the function actually use `this` in its function body. For example:
 
 ```js
-var foo = function() {
-  console.log("not using this");
-}.bind(bar)
+var boundGetName = (function getName() {
+    return this.name;
+}).bind({ name: "ESLint" });
+
+console.log(boundGetName());      // "ESLint"
 ```
 
-## Rule details
+This code is an example of a good use of `bind()` for setting the value of `this`.
 
-The following patterns are considered warnings:
+Sometimes during the course of code maintenance, the `this` value is removed from the function body. In that case, you can end up with a call to `bind()` that doesn't accomplish anything:
 
 ```js
-/*eslint no-extra-bind: 2*/
```

Comment:
This is diff is deceiving, the comments are still there in the actual code examples for the rule. This code block is in the introduction.